tests/phpunit/tests/functions/cleanDirsizeCache.php

```diff
@@ -35,19 +35,19 @@ public function data_clean_dirsize_cache_with_invalid_inputs() {
 		return array(
 			'null'         => array(
 				'path'             => null,
-				'expected_message' => '<code>clean_dirsize_cache()</code> only accepts a non-empty path string, received <code>NULL</code>.',
+				'expected_message' => 'clean_dirsize_cache(): <code>clean_dirsize_cache()</code> only accepts a non-empty path string, received <code>NULL</code>.',
```

Comment:
As suggested above, adding the function name is redundant as the error message already has it built in.

```suggestion
				'expected_message' => '<code>clean_dirsize_cache()</code> only accepts a non-empty path string, received <code>NULL</code>.',
```